### ISSUE TITLE
Configuration vs. Annotation

### ISSUE THREAD

**Issue by colonelclick** [DamienHarper/auditor-bundle#196]
I was trying to use configuration and I received no errors but it did not work*, so I had to switch to annotation.

*I added entries to dh_doctrine_audit.yaml (below) but there was no diff detected for a doctrine migration, and the entities were not logged by my application, but no errors were generated.

However, as soon as I added the annotation for `* @Audit\Auditable` into the Customer entity shown in the yaml below a diff was able to be generated and the entity was correctly logged.

Is this expected? Or should it work based purely on configuration?

```
dh_doctrine_audit:
    ignored_columns:
        - createdAt
        - updatedAt
    entities:
        App/Entity/Customer: ~
```

**Comment by DamienHarper** [comment]
Hi @colonelclick Thanks for the report. BTW, configuration should work as you expected.

Could you please post the following:
* the version of the bundle
* the PHP version
* the Symfony version

Did you try once it worked with annotations to remove them but keep configuration? Did it worked?

Was your schema updated? (cf https://github.com/DamienHarper/DoctrineAuditBundle/blob/master/doc/60-troubleshooting.md#ive-added-an-new-entity-in-the-config-file-but-its-not-audited)

**Comment by DamienHarper** [comment]
@colonelclick Are you still facing an issue?

**Comment by colonelclick** [comment]
Hi Damien, thank you for following up on this. Yes I will finish this bug report, I got a little busy this week, I apologize for the delay.

**Comment by colonelclick** [comment]
php: 7.2.23
symfony: 4.4.0
dhBundle: 3.4.1

I tried removing the annotation and restoring the yaml. When I did that, no changes were tracked, and the /audit screen reported there were 0 audited entities. 

Reversing this procedure restored correct function.

Doctrine does not generate a diff if I modify the yaml, but it does generate a diff when I add the annotation to a new entity.

I can tell the yaml is being read because it is correctly ignoring my ignored_columns key. Even if I put a fake entity in the entities section, it does not throw an error for invalid entity.
